addons/account/models/account_move.py

```diff
@@ -3964,6 +3972,10 @@ def action_open_business_doc(self):
             'target': 'current',
         }
 
+    def action_update_fpos_values(self):
+        self.invoice_line_ids._compute_tax_ids()
+        self.invoice_line_ids._compute_account_id()
```

Comment:
Also need to map the payable/receivable accounts, as well as the tax accounts
```suggestion
        self.line_ids._compute_account_id()
```